### ISSUE TITLE
Occupancy Status

### ISSUE THREAD

**Issue by aaronbrethorst** [OneBusAway/OBAKit#34]
~We should first wait for the current efforts around improving this functionality at the GTFS spec level to come to rest, and then implement whatever is defined there after it exits the experimental stage.~

* Proposal: Support multi-car crowding in GTFS-RT - https://github.com/google/transit/pull/237
* Proposal: GTFS-MeanOccupancies - https://github.com/google/transit/pull/240

This is ready to go. Needs a design, but otherwise can be worked on.

Output for `api/where/arrivals-and-departures-for-stop/1_8407.json`: 

[1_8407.json.zip](https://github.com/OneBusAway/OBAKit/files/5797653/1_8407.json.zip)

Validation can be done on WMATA's servers.

Corresponding Android issue: https://github.com/OneBusAway/onebusaway-android/issues/1059

**Comment by aaronbrethorst** [comment]
Output for `api/where/arrivals-and-departures-for-stop/1_8407.json`

[1_8407.json.zip](https://github.com/OneBusAway/OBAKit/files/5797653/1_8407.json.zip)

**Comment by barbeau** [comment]
@aaronbrethorst Here's what the OBA Android UI looks like with live WMATA data (designed way back in 2018) using new code in WIP PR https://github.com/OneBusAway/onebusaway-android/pull/1060:

![image](https://user-images.githubusercontent.com/928045/104244666-ead62c00-5430-11eb-9653-48da345dc0d7.png)

For WMATA, MANY_SEATS_AVAILABLE is 1 person icon, FEW_SEATS_AVAILABLE is 2 person icons, and FULL is 3 person icons. 

Note that this may need to be adjusted for other agencies depending on which GTFS-RT OccupancyStatus enums they choose to publish (this is a [whole thing](https://github.com/google/transit/pull/259) in the spec still).

To keep the UI from looking too much like a Christmas tree, I'm using green for shading the 1 and 2 person icons, and red for the 3 person icons. I tried adding a yellow for 2 persons back when I first implemented this, but the color on the screen really starts to get overwhelming. Historical occupancy (not published by WMATA) would show up as gray, just like scheduled arrivals. There is an "About occupancy" menu item when you tap on each arrival list item available that explains this.
